sample/dm_databases.yaml

```diff
@@ -0,0 +1,49 @@
+version: '2.3'
+
+services:
+    dm_upstream_mysql:
+        image: mysql:5.7
+        container_name: dm_upstream_mysql
+        ports:
+            - "3306:3306"
+        volumes:
+            - ./config/mysql.cnf:/etc/mysql/conf.d/mysql.cnf
+        environment:
+            MYSQL_ROOT_PASSWORD: 123456
+
+    dm_downstream_pd:
+        image: pingcap/pd:v5.4.0
+        ports:
+            - "2379"
+        command:
+            - --name=dm_downstream_pd
+            - --client-urls=http://0.0.0.0:2379
+            - --peer-urls=http://0.0.0.0:2380
+            - --advertise-client-urls=http://dm_downstream_pd:2379
+            - --advertise-peer-urls=http://dm_downstream_pd:2380
+            - --initial-cluster=dm_downstream_pd=http://dm_downstream_pd:2380
+            - --data-dir=/data/dm_downstream_pd
+        restart: on-failure
+
+    dm_downstream_tikv:
+        image: pingcap/tikv:v5.4.0
+        command:
+            - --addr=0.0.0.0:20160
+            - --advertise-addr=dm_downstream_tikv:20160
+            - --pd=dm_downstream_pd:2379
+        depends_on:
+            - "dm_downstream_pd"
+        restart: on-failure
+
+    dm_downstream_tidb:
```

Comment:
in fact we can use TiDB with unistore (no TiKV)

Reply:
Yep, I concern unistore is not a completed implementation with new features (or new feature is added later than TiKV), so I use TiKV here.